Sources/Spezi/Spezi.docc/Spezi Ecosystem.md

```diff
@@ -0,0 +1,268 @@
+
+> Tip: Check out the [SpeziFHIR Documentation](https://swiftpackageindex.com/StanfordSpezi/SpeziFHIR/documentation/spezifhir) for implementation details.
+
+
+### LLMs and AI Integration
+This module includes three submodules:
+
+- The **[SpeziChat](https://github.com/StanfordSpezi/SpeziChat)** module implements sophisticated chat-based user interfaces with comprehensive message history management and seamless AI integration for natural language processing.
+
+@Row{
+    @Column {
+        ![Screenshot displaying the regular chat view.](https://raw.githubusercontent.com/StanfordSpezi/SpeziChat/main/Sources/SpeziChat/SpeziChat.docc/Resources/ChatView.png)
+    }
+    @Column {
+        ![Screenshot displaying the text input chat view.](https://raw.githubusercontent.com/StanfordSpezi/SpeziChat/main/Sources/SpeziChat/SpeziChat.docc/Resources/ChatView+TextInput.png)
+    }
+    @Column {
+        ![Screenshot displaying the voice input chat view.](https://raw.githubusercontent.com/StanfordSpezi/SpeziChat/main/Sources/SpeziChat/SpeziChat.docc/Resources/ChatView+VoiceInput.png)
+    }
+}
+
+- The **[SpeziSpeech](https://github.com/StanfordSpezi/SpeziSpeech)** module delivers comprehensive voice interaction capabilities, offering both speech synthesis and recognition features along with robust session management for voice input and output.
+
+- The **[SpeziLLM](https://github.com/StanfordSpezi/SpeziLLM)** module integrates large language models to provide AI-driven functionalities, enabling sophisticated text generation and contextual understanding for complex queries.
+
+@Row{
+    @Column {
+        ![Screenshot displaying the Chat View utilizing the OpenAI API from SpeziLLMOpenAI.](https://raw.githubusercontent.com/StanfordSpezi/SpeziLLM/main/Sources/SpeziLLMOpenAI/SpeziLLMOpenAI.docc/Resources/ChatView.png)
+    }
+    @Column {
+        ![Screenshot displaying the Local LLM Download View from SpeziLLMLocalDownload.](https://raw.githubusercontent.com/StanfordSpezi/SpeziLLM/main/Sources/SpeziLLMLocalDownload/SpeziLLMLocalDownload.docc/Resources/LLMLocalDownload.png)
+    }
+    @Column {
+        ![Screenshot displaying the Chat View utilizing a locally executed LLM via SpeziLLMLocal.](https://raw.githubusercontent.com/StanfordSpezi/SpeziLLM/main/Sources/SpeziLLMLocal/SpeziLLMLocal.docc/Resources/ChatView.png)
+    }
+}
+
+> Tip: 
+Check out [SpeziChat Documentation](https://swiftpackageindex.com/StanfordSpezi/SpeziChat/documentation/spezichat), [SpeziSpeech Documentation](https://swiftpackageindex.com/StanfordSpezi/SpeziSpeech/1.1.1/documentation/spezispeechrecognizer) for implementation details.
+and [SpeziLLM Documentation](https://swiftpackageindex.com/StanfordSpezi/SpeziLLM/documentation/spezillm) for implementation details.
```

Comment:
Combining modules together then leads to these side effects where tons of doc refs are within one block. We should keep that in mind when making a decision on how to structure the doc